release/models/cfm/openconfig-cfm-types.yang

```diff
@@ -0,0 +1,392 @@
+module openconfig-cfm-types {
+
+  yang-version "1";
+
+  // namespace
+  namespace "http://openconfig.net/yang/cfm-types";
+
+  prefix "oc-cfm-types";
+
+  // import some basic types
+  import ietf-yang-types { prefix yang; }
```

Comment:
s/ietf-yang-types/openconfig-yang-types/

And if types do not exist in the OC types you need then they may very well need an OC flavor to be defined

Reply:
hello earies, the import is unused and removed. thanks for the review!